### PR TITLE
Docs/optimizely x plugin

### PR DESCRIPTION
**Pull request by shoota**
# TL;DR

### How to check

*   checkout this pr.
*   yarn install
*   yarn start

### Check list

*   [ ] check [new page](http://localhost:3000/docs/en/optimizely_x-plugin.html)

### CODE REVIEW

File: examples/index.pug [hunk 1]
```diff
@@ -47,10 +47,11 @@ html
       .panel.panel-default
```
Comment (shoota) [inline examples/index.pug]: tweak page list and theirs grouping

File: docs/optimizely_x-plugin.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+---
```
Comment (shoota) [inline docs/optimizely_x-plugin.md]: new page

File: docs/optimizely_x-plugin.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+The Optimizely X plugin works A/B testing integration for Optimizely X.
+
+This plugin will send A/B testing informaiton as [Event data](./field-reference.html#event-tracking).
+
+| Field         | Assigned Value                   |
+| :------------ | :------------------------------- |
+| eventCategory | Constant string as 'optimizely'. |
+| eventAction   | The experiment ID string in VWO. |
```
Comment (heavenshell) [inline docs/optimizely_x-plugin.md]: `Optimizely X` ?

File: docs/optimizely_x-plugin.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+| eventLabel    | The variation ID for renderd.    |
```
Comment (heavenshell) [inline docs/optimizely_x-plugin.md]: `s/renderd/rendered` ?

File: docs/optimizely_x-plugin.md (hunk 3 above)
Comment (shoota) [reply docs/optimizely_x-plugin.md]: Oh...

**Pull request merged**